app/services/hyrax/caching_iiif_manifest_builder.rb

```diff
@@ -45,13 +45,9 @@ def manifest_cache_key(presenter:)
       end
 
       ##
-      # @note `etag` is a better option than the solr document `_version_`; the
-      #   latter isn't always available, depending on how the presenter was
-      #   built!
-      #
       # @return [String]
       def version_for(presenter)
-        presenter.etag
+        presenter.version
```

Comment:
I like this explicit method name usage; Instead of relying on an implicit understanding of `etag` (which is reasonable but just a bit less clear).